install.md

```diff
@@ -0,0 +1,48 @@
+# conmon Installation Instructions
+
+In most cases, conmon should come packaged with your favorite container manager.
+However, if you'd like to try building it from source, follow the steps below.
+
+## Fedora, CentOS, RHEL, and related distributions:
+
+```bash
+sudo yum install -y \
+  gcc \
+  git \
+  glib2-devel \
+  glibc-devel \
+  make \
+  pkgconfig \
+  runc
+```
+
+## Debian, Ubuntu, and related distributions:
+
+```bash
+sudo apt-get install \
+  gcc \
+  git \
+  libc6-dev \
+  libglib2.0-dev \
+  pkg-config \
+  make \
+  runc
+```
+
+## Clone the repository and compile:
+
+``` bash
+git clone https://github.com/containers/conmon.git
+make
+```
+
+
+## Install to the directory you want:
+There are three options for installation, depending on your need. Each can have the PREFIX overridden. The PREFIX defaults to `/usr/local`
```

Comment:
s/your need/your environment/

Reply:
Would it be safe to add to the end here "`/usr/local` for most Linux distributions."  Regardless, please add an ending period.